### DISCUSSION TITLE
Internationalization for document (i18n)

### DISCUSSION THREAD

**Discussion by erfan-goodarzi** [TanStack/router#1406] [Ideas]
Hello, there. So I was reading the tanstack/router document and realized that it would be great if we could have it in another languages. It can help people grasp concepts better. 

what do u think about this idea? If you agree, I can start integrating with i18n. Please feel free to remark, whether you agree or disagree.

**Comment by tiavina-mika** [comment]
Any news on this?
We can add internationalization in code based routes, but how to do that in file based routes?

**Comment by aymericzip** [comment]
If it can help:
https://intlayer.org/doc/environment/tanstack-start
